03-transactions.md

```diff
@@ -234,6 +235,45 @@ The witness script for the output is:
 
 To spend this via penalty, the remote node uses a witness stack `<revocationsig> 1` and to collect the output the local node uses an input with nSequence `to_self_delay` and a witness stack `<local_delayedsig> 0`
 
+## Closing Transaction
+
+Note that there are two possible variants for each node.
+
+* version: 1
```

Comment:
@roasbeef said version 1, since version 2 only needed for CLTV, which we don't need here.  I don't really mind...